lib/src/upgrader.dart

```diff
@@ -299,6 +299,8 @@ class Upgrader with WidgetsBindingObserver {
     bool rv = true;
     if (state.debugDisplayAlways || (state.debugDisplayOnce && !_hasAlerted)) {
       rv = true;
+    } else if (forceUpgrade) {
```

Comment:
@minhdanh How is using `forceUpgrade ` any different from `debugDisplayAlways `, other than the name?

Reply:
Hi, I've just tested with `debugDisplayAlways` and could create the same effect. So I'm closing this now. Thank you!